usr/bin/js/openWeather.js

```diff
@@ -74,11 +76,17 @@ function getOpenWeather (location, tempUnit, distanceUnit) {
             else {
                 weather='cloudy';
             }
+	var datenow = new Date();
+	var timenow = Math.floor(datenow.getTime()/1000);
+	var isNight = (timenow < jsonResult.sys.sunrise || timenow > jsonResult.sys.sunset);
+
+
 
             var weatherObj = {
                 'temperature': tempVal,
                 'condition': weather,
-                'wind': windSpeed
+                'wind': windSpeed,
+		'isNight': isNight
```

Comment:
also can you align the properties. I probably should have also aligned the values...

Reply:
I think I updated it.